go.mod

```diff
@@ -1,79 +1,118 @@
 module github.com/smartcontractkit/chainlink-ccip
 
-go 1.24
+go 1.24.1
 
 require (
 	github.com/deckarep/golang-set/v2 v2.6.0
 	github.com/ethereum/go-ethereum v1.15.3
 	github.com/patrickmn/go-cache v2.1.0+incompatible
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.20.0
+	github.com/prometheus/client_golang v1.22.0
 	github.com/prometheus/client_model v0.6.1
 	github.com/smartcontractkit/chain-selectors v1.0.47
-	github.com/smartcontractkit/chainlink-common v0.4.2-0.20250121163309-3e179a73cb92
+	github.com/smartcontractkit/chainlink-common v0.7.1-0.20250421201900-888b361327e8
+	github.com/smartcontractkit/chainlink-evm v0.0.0-20250422160140-830158ed5ba8
```

Comment:
My preference would be to not depend on chainlink-evm, is there a reason the compile script has to be there if we're moved to product-specific repos at this point?

Reply:
I copy/pasted the template that was the only thing I was using from `chainlink-evm`. We can keep it this way and refactor it later into a new repo if necessary